tests/vmi_multus_test.go

```diff
@@ -834,15 +834,15 @@ var _ = Describe("[Serial]SRIOV", func() {
 			// it's hard to match them.
 		})
 
-		// pingThroughSriov instantiates two VMs connected through SR-IOV and
-		// pings between them.
+		// createSriovVMs instantiates two VMs connected through SR-IOV.
 		// Note: test case assumes interconnectivity between SR-IOV
 		// interfaces. It can be achieved either by configuring the external switch
 		// properly, or via in-PF switching for VFs (works for some NIC models)
-		pingThroughSriov := func(cidrA, cidrB string) {
+		createSriovVMs := func(cidrA, cidrB string) (*v1.VirtualMachineInstance, *v1.VirtualMachineInstance) {
```

Comment:
it also set IPs, that only those two test do, so we cant.
i just splitted the code, in order to move the skip to later stage,
once the PR that fix #4432 is merged we can combine the functions back, or even leave them splitted because the split is logical

Reply:
sounds good.